x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -183,6 +206,60 @@ export class LoginPage extends Component<Props, State> {
       );
     }
 
+    if (!isCookiesEnabled) {
+      if (isWindowEmbedded()) {
+        return (
+          <div style={{ maxWidth: '36em', margin: 'auto', textAlign: 'center' }}>
+            <EuiText color="subdued">
+              <p>
+                {this.props.sameSiteCookies !== 'None' ? (
+                  <FormattedMessage
+                    id="xpack.security.loginPage.openInNewWindowOrChangeKibanaConfigTitle"
+                    defaultMessage="To view this content, open it in a new window or ask your administrator to allow cross-origin cookies."
+                  />
+                ) : (
+                  <FormattedMessage
+                    id="xpack.security.loginPage.openInNewWindowOrChangeBrowserSettingsTitle"
+                    defaultMessage="To view this content, open it in a new window or adjust your browser settings to allow third-party cookies."
+                  />
+                )}
+              </p>
+            </EuiText>
+            <EuiSpacer />
+            <EuiButton
+              href={window.location.href}
+              target="_blank"
+              iconType="popout"
+              iconSide="right"
+              fill
+            >
+              <FormattedMessage
+                id="xpack.security.loginPage.openInNewWindowButton"
+                defaultMessage="Open in new window"
+              />
+            </EuiButton>
+          </div>
+        );
+      }
+
+      return (
+        <DisabledLoginForm
+          title={
+            <FormattedMessage
+              id="xpack.security.loginPage.requiresCookiesTitle"
+              defaultMessage="Cookies are required to log into Elastic"
```

Comment:
__nit:__? (as per https://github.com/elastic/kibana/pull/120944#discussion_r768924950)
```suggestion
              defaultMessage="Cookies are required to log in to Elastic"
```